tests/python_tests/test_llm_pipeline.py

```diff
@@ -451,6 +451,17 @@ def test_perf_metrics(generation_config, prompt):
     assert len(raw_metrics.m_batch_sizes) > 0
     assert len(raw_metrics.m_durations) > 0
 
+
+@pytest.mark.precommit
+@pytest.mark.nightly
+def test_eos_token_with_property_passes_config_validation():
+    ov_pipe = read_model(('katuni4ka/tiny-random-phi3', Path('tiny-random-phi3')))[4]
+    ov_pipe.generate(
+        "",
+        max_new_tokens=1,
+        eos_token_id=ov_pipe.get_generation_config().eos_token_id + 1
+    )
```

Comment:
we don't need pipeline to test generation config.

I propose to extend this test to check that set values as actually the same as kwargs:
https://github.com/openvinotoolkit/openvino.genai/blob/ce55429d5e1350b980506a1afb81ce93c2ca6de8/tests/python_tests/test_generation_config.py#L50-L56
with extra check that `stop_token_ids` contains `eos_token_id`